docs_src/OVMS/capiPipelineRun.md

```diff
@@ -0,0 +1,92 @@
+
+| EV Name                   | Description                                           |
+| --------------------------| ------------------------------------------------------|
+| RENDER_PORTRAIT_MODE      | value 1 for rendering in portrait mode                |
+| GST_DEBUG                 | value 1 for running GStreamer in debug mode           |
+| USE_ONEVPL                | value 1 for using OneVPL CPU & GPU Support            |
+| PIPELINE_EXEC_PATH        | pipeline execution path inside container              |
+| GST_VAAPI_DRM_DEVICE      | GStreamer VAAPI DRM device input                      |
+| TARGET_GPU_DEVICE         | privilege mode for using GPU devices if any           |
+| LOG_LEVEL                 | [log level](https://gstreamer.freedesktop.org/documentation/tutorials/basic/debugging-tools.html?gi-language=c#the-debug-log) to be set when running gst pipeline         |
+| RENDER_MODE               | option to display the input source video stream with the inferencing results              |
+| cl_cache_dir              | cache directory in container                          |
+
+Here is the value we've set so far for capi_face_detection.env
+```text
+RENDER_PORTRAIT_MODE=0
+GST_DEBUG=0
+USE_ONEVPL=1
+PIPELINE_EXEC_PATH=pipelines/face_detection/face_detection
+GST_VAAPI_DRM_DEVICE=/dev/dri/renderD128
+TARGET_GPU_DEVICE=--privileged
+LOG_LEVEL=0
+RENDER_MODE=1
+cl_cache_dir=/home/intel/gst-ovms/.cl-cache
+```
+
+## Add A Profile Launcher Configuration File
+The details about Profile Launcher configuration can be found [here](./profileLauncherConfigs.md), below is detail of capi face detection profile launcher configuration, located at `configs/opencv-ovms/cmd_client/res/capi_face_detection/configuration.yaml`.
+```yaml
+OvmsSingleContainer: true
+OvmsClient:
+  DockerLauncher:
+    Script: docker-launcher.sh
+    DockerImage: openvino/model_server-capi-gst-ovms:latest
+    ContainerName: capi_face_detection
+    Volumes:
+      - "$cl_cache_dir:/home/intel/gst-ovms/.cl-cache"
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - "$RUN_PATH/sample-media/:/home/intel/gst-ovms/vids"
+      - "$RUN_PATH/configs/opencv-ovms/gst_capi/extensions:/home/intel/gst-ovms/extensions"
+      - "$RUN_PATH/results:/tmp/results"
+      - "$RUN_PATH/configs/opencv-ovms/models/2022/:/home/intel/gst-ovms/models"
+  PipelineScript: ./run_face_detection.sh
+  PipelineInputArgs: "" # space delimited like we run the script in command and take those input arguments
+  EnvironmentVariableFiles:
+    - capi_face_detection.env
+```
+
+## Build and Run
+Here are the quick start steps to build and run OVMS C API face detection pipeline profile:
+1. Build gst-capi ovms with profile-launcher: `make build-gst-capi`
+2. Download sample video files: `cd benchmark-scripts/ && ./download_sample_videos.sh && cd ..`
+3. Start simulator camera: `make run-camera-simulator`
```

Comment:
```suggestion
## Build and Run

Here are the quick start steps to build and run OVMS C API face detection pipeline profile:

1. Build gst-capi ovms with profile-launcher: `make build-gst-capi`
2. Download sample video files: `cd benchmark-scripts/ && ./download_sample_videos.sh && cd ..`
3. Start simulator camera: `make run-camera-simulator`
```
please insert a new line for between title and the first sentence and then also a new line before bullets or number in order for mkdocs generation properly